scripts/avatars/trops_avatars_seeding.py

```diff
@@ -0,0 +1,164 @@
+import os
+
+from great_ape_safe import GreatApeSafe
+from helpers.addresses import r
+from pycoingecko import CoinGeckoAPI
+
+
+cg = CoinGeckoAPI(os.getenv("COINGECKO_API_KEY"))
+
+# dollar denominated
+FUNDS_PER_POSITION = 20_000
+
+
+def bpt_ratios_amounts(bpt, token0_rate, token1_rate, token0_decimals, token1_decimals):
+    weights = bpt.getNormalizedWeights()
+    ratio0 = weights[0] / 1e18
+    ratio1 = weights[1] / 1e18
+
+    return (
+        int((FUNDS_PER_POSITION * ratio0 / token0_rate) * 10 ** token0_decimals),
+        int((FUNDS_PER_POSITION * ratio1 / token1_rate) * 10 ** token1_decimals),
+    )
+
+
+def seed():
+    trops = GreatApeSafe(r.badger_wallets.treasury_ops_multisig)
+    trops.init_balancer()
+    trops.init_curve()
+
+    # assets
+    badger = trops.contract(r.treasury_tokens.BADGER)
+    wbtc = trops.contract(r.treasury_tokens.WBTC)
+    digg = trops.contract(r.treasury_tokens.DIGG)
+    gravi = trops.contract(r.sett_vaults.graviAURA)
+    weth = trops.contract(r.treasury_tokens.WETH)
+    usdc = trops.contract(r.treasury_tokens.USDC)
+    frax = trops.contract(r.treasury_tokens.FRAX)
+    reth = trops.contract(r.treasury_tokens.RETH)
+
+    # snap
+    trops.take_snapshot(tokens=[badger, wbtc, weth, usdc])
+
+    # decimals
+    badger_decimals = badger.decimals()
+    wbtc_decimals = wbtc.decimals()
+    reth_decimals = reth.decimals()
+
+    # balancer bpts
+    bpt_wethreth = trops.contract(r.balancer.B_50_WETH_50_RETH)
+    bpt_badgerwbtc = trops.contract(r.balancer.B_20_BTC_80_BADGER)
+    bpt_badgerreth = trops.contract(r.balancer.B_50_BADGER_50_RETH)
+    bpt_wbtcdigggravi = trops.contract(r.balancer.bpt_40wbtc_40digg_20graviaura)
+
+    # convex lps
+    pool_fraxbp = trops.contract(r.crv_factory_pools.badgerFRAXBP_f)
+    badger_fraxbp = trops.contract(r.treasury_tokens.badgerFRAXBP_f_lp)
+    wcvx_badger_fraxbp = trops.contract(r.convex.frax.wcvx_badger_fraxbp)
+    fraxbp_zap = trops.contract(r.curve.zap_fraxbp)
+
+    # pricing
+    prices = cg.get_price(
+        ids=["badger-dao", "wrapped-bitcoin", "weth", "rocket-pool-eth"],
+        vs_currencies="usd",
+    )
+    badger_rate = prices["badger-dao"]["usd"]
+    wbtc_rate = prices["wrapped-bitcoin"]["usd"]
+    weth_rate = prices["weth"]["usd"]
+    reth_rate = prices["rocket-pool-eth"]["usd"]
+
+    # avatars
+    aura_avatar = trops.contract(r.avatars.aura)
+    convex_avatar = trops.contract(r.avatars.convex)
+
+    # aura avatar deposits
+    trops.balancer.deposit_and_stake(
+        [wbtc, badger],
+        bpt_ratios_amounts(
+            bpt_badgerwbtc, wbtc_rate, badger_rate, wbtc_decimals, badger_decimals
+        ),
+        pool=bpt_badgerwbtc,
+        stake=False,
+    )
+
+    weth_mantissa = int(((FUNDS_PER_POSITION // 2) / weth_rate) * 1e18 / 0.98)
```

Comment:
unsure why i did that type of division, but it is not needed, removed in 0fcb6fb3d541ea962d74e9ed93d02783e7567a16

the *.98* is for the opposite of dusty, since it is dividing. its to get a bit more of a balance just to make sure that we have enough later for the swap